builtin/mainmenu/dlg_contentstore.lua

```diff
@@ -353,7 +352,8 @@ function store.get_formspec(dlgdata)
 			"container[0.375,0.375]",
 			"field[0,0;7.225,0.8;search_string;;", core.formspec_escape(search_string), "]",
 			"field_close_on_enter[search_string;false]",
-			"button[7.225,0;2,0.8;search;", fgettext("Search"), "]",
+            		"image_button[7.3,0;0.8,0.8;", core.formspec_escape(defaulttexturedir .. "clear.png"), ";clear;]",
```

Comment:
mixed indentation

this line begins with 12 spaces and 2 tabs, it should be 4 tabs